examples/source_map_support/defs.bzl

```diff
@@ -7,22 +7,32 @@ Use them wherever you would use rules_js's `js_binary` and `js_test`.
 
 load("@aspect_rules_js//js:defs.bzl", _js_binary = "js_binary", _js_test = "js_test")
 
-def js_binary(data = [], node_options = [], **kwargs):
+def js_binary(data = [], chdir = None, node_options = [], **kwargs):
+    rel = "./"
+    if chdir:
+        rel = rel + "/".join([".." for _ in chdir.split("/")]) + "/"
```

Comment:
can you use https://docs.aspect.build/rules/aspect_bazel_lib/docs/paths/#to_rlocation_path instead of writing new pathing logic?

Reply:
That requires `ctx` so it has to be in a rule implementation?

I figured this is a test and not worth much more effort 🤷

Reply:
sure, the bar is lower here in the example code, but it does look a little hacky. I don't care enough to ask you to spend more time